catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/InspectCommandTest.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.commands.catalog;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.StringContains.containsString;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import ddf.catalog.CatalogFramework;
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.Result;
+import ddf.catalog.filter.proxy.builder.GeotoolsFilterBuilder;
+import ddf.catalog.operation.QueryResponse;
+import org.junit.Test;
+
+public class InspectCommandTest extends CommandCatalogFrameworkCommon {
+
+  @Test
+  public void testMultivaluedAttr() throws Exception {
+    Metacard metacard = getMockMultiValuedMetacard();
+
+    // mock out catalog framework
+    QueryResponse mockQueryResponse = mock(QueryResponse.class);
+    Result mockResult = mock(Result.class);
+    doReturn(metacard).when(mockResult).getMetacard();
+    doReturn(ImmutableList.of(mockResult)).when(mockQueryResponse).getResults();
+
+    CatalogFramework mockCatalogFramework = mock(CatalogFramework.class);
+    doReturn(mockQueryResponse).when(mockCatalogFramework).query(any());
+
+    InspectCommand command = new InspectCommand();
+    command.catalogFramework = mockCatalogFramework;
+    command.id = "id";
+    command.filterBuilder = new GeotoolsFilterBuilder();
+
+    command.executeWithSubject();
+
+    String consoleOutput = CommandCatalogFrameworkCommon.consoleOutput.getOutput();
```

Comment:
I don't think the `CommandCatalogFrameworkCommon` is necessary because this test inherits from that class.

Reply:
And you know I swore that, too. But Intellij disagreed. Probably just needed to rebuild the module but I just did this because I was moving fast. I'll look into it

Reply:
It was because my instance var was named the same as that global so it was ambiguous 😂